docs/docs/steps/actions/README.md

```diff
@@ -0,0 +1,126 @@
+# Actions (legacy)
```

Comment:
This looks like another file that was included from v1. Was this file potentially still in your local directory, but then re-included with a `git add`?